puppetboard/templates/_macros.html

```diff
@@ -79,6 +79,9 @@
     {% else -%}
     {
       "targets": -1,
+      {% if natural_time_delta_sort %}
```

Comment:
I think that this should go above, under the case where `table_html_id == 'facts_table'` as we are changing the sort in that table.